packages/blaze/view.js

```diff
@@ -188,6 +188,40 @@ Blaze.View.prototype.autorun = function (f, _inViewScope) {
   return c;
 };
 
+Blaze.View.prototype._errorIfShouldntCallSubscribe = function () {
+  var self = this;
+
+  if (! self.isCreated) {
+    throw new Error("View#subscribe must be called from the created callback at the earliest");
+  }
+  if (self._isInRender) {
+    throw new Error("Can't call View#subscribe from inside render(); try calling it from the created or rendered callback");
+  }
+  if (self.isDestroyed) {
+    throw new Error("Can't call View#subscribe from inside the destroyed callback, try calling it inside created or rendered.");
+  }
+};
+
+/**
+ * Just like Blaze.View#autorun, but with Meteor.subscribe instead of
+ * Tracker.autorun. Stop the subscription when the view is destroyed.
+ * @return {SubscriptionHandle} A handle to the subscription so that you can
+ * see if it is ready, or stop it manually
+ */
+Blaze.View.prototype.subscribe = function (/* arguments */) {
+  var self = this;
+
+  self._errorIfShouldntCallSubscribe();
+
+  var subHandle = Meteor.subscribe.apply(Meteor, arguments);
```

Comment:
Sure, as long as you expect that the answer is going to be something like "add another function with a different name" and not "I sure wish I had designed this function to take an `options` argument but I can't now".

The obvious case to me is — are we going to want to use this behavior in `accounts-ui`?  Because that's going to want to use `Accounts.connection`.